easybuild/easyconfigs/l/libav/libav-11.10-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,33 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'libav'
+version = '11.10'
+
+homepage = 'https://libav.org/'
+
+description = """
+ Libav is a friendly and community-driven effort to provide its users with
+ a set of portable, functional and high-performance libraries for dealing
+ with multimedia formats of all sorts.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+# https://libav.org/releases/libav-11.9.tar.gz
```

Comment:
@JackPerdue please remove this outdated comment

Reply:
Done